src/app/search/facet-search/facet-search-update.pipe.ts

```diff
@@ -0,0 +1,28 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'getFacetSearchUpdate',
+})
+/**
+ * This pipe updates the number of hidden items in the facet when a search is completed
+ *
+ * @param {Array<any>} items The items in the facet
+ * @param {string} searchText The search text entered
+ * @returns {string} The numbers of hidden items in the facet
+ * @memberof GetFacetSearchResultsPipe
+ */
+export class GetFacetSearchUpdatePipe implements PipeTransform {
+  transform(unfilteredItems: Array<any>, searchText: string): string {
+    let hiddenFilteredSize;
+    if (!searchText && unfilteredItems.length > 5) {
+      hiddenFilteredSize = unfilteredItems.length - 5;
+      return hiddenFilteredSize.toString() + ' more';
+    }
+    const filteredSize = unfilteredItems.filter((item) => item.toLowerCase().includes(searchText)).length;
```

Comment:
Shouldn't you make a lower-case copy of searchText at the beginning of this method? Otherwise you're relying on the caller always passing in lower-case.